### ISSUE TITLE
No error when providing more than one unpacked arbitrary-length tuple argument

### ISSUE THREAD

**Issue by Viicos** [microsoft/pyright#10219]
This doesn't seem to be explicitly specified in the spec, but the following raises at runtime:

Code sample in [pyright playground](https://pyright-play.net/?pythonVersion=3.13&strict=true&code=MYGwhgzhAECCDaAqAKhAugLmgOlwWAChCFEAXAVwAcQBTeASwDtSAaHXNNsq2h5t3NjRpo0AMTRGAe2g0ATnKly2AMzD0QMMKWhzyzegFsaGQhOQBPSjQCiCpVgCySmtFIALMI2hTGr-ZRgwADWNAAm0GByAEb0pHJRFgC0tIwA5h5uPK5RaeTGzISE0VgU1HTc5Xys7EJcZbxMNYLCohLyisqRMDQAHtbApOGEQA)

```python
class A[*Ts]: ...

A[*tuple[int, ...], *tuple[int, ...]]  # no error, fails at runtime:
# TypeError: More than one unpacked arbitrary-length tuple argument

b: tuple[*tuple[int, ...], *tuple[int, ...]]  # error, as expected

```

**Comment by erictraut** [comment]
Yeah, I think this should be flagged as illegal by a type checker.

Interestingly, mypy crashes with this piece of code. Might be worth filing a bug on that.

**Comment by erictraut** [comment]
I filed a bug in the [mypy issue tracker](https://github.com/python/mypy/issues/18856).

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.399.
